### ISSUE TITLE
Java Object transformation to list

### ISSUE THREAD

**Issue by albilu** [datafaker-net/datafaker#979]
Is Java object transformation to list possible??

From the documentation here https://www.datafaker.net/documentation/schemas/#java-object-transformation it seems only singleton is posssible unless i missed something.

Please advise. Here is my iomplementation:

```java
Faker faker = new Faker();

Schema<Object, ?> schema = Schema.of(
                Field.field("name", () -> faker.name().firstName()),
                Field.field("email", () -> faker.internet().emailAddress())
        );

BaseFaker bfaker = new BaseFaker();
Client client = bfaker.populate(Client.class, schema); //Here i would like to have a collection of fake Clients instead
```

**Comment by snuyanzin** [comment]
you could use `FakeCollection` or `FakeSequence` https://www.datafaker.net/documentation/sequences/
to do it like 
```java
List<DefaultPerson> list = new FakeCollection.Builder<DefaultPerson>()
            .suppliers(() -> Faker.populate(DefaultPerson.class)).generate();

        System.out.println(list);
```

**Comment by albilu** [comment]
@snuyanzin Thanks for the feedback.
Issue now is the Client objects  have the same fields values. Goal is to have random value for each instance.

```java
List<Client> client = new FakeCollection.Builder<Client>().suppliers(()
                -> Faker.populate(Client.class, schema))
                .len(5).generate();

client2.forEach((t) -> {
            System.out.println(t.getName() + " " + t.getEmail());

        });
```
Output
```
Hellen cherrie.zieme@yahoo.com
Hellen cherrie.zieme@yahoo.com
Hellen cherrie.zieme@yahoo.com
Hellen cherrie.zieme@yahoo.com
```

Although it works with simple Collection generation 
```java
List<String> names = 
    faker.collection(
            () -> faker.name().firstName(), 
            () -> faker.name().lastName())
        .len(5)
        .generate();
```
Output
```
Corkery
Otha
Moore
Tisa
Sawayn
```

**Comment by snuyanzin** [comment]
Can you share the complete code snippet?

**Comment by bodiam** [comment]
Hi @albilu , could you give the snapshot release a try?

**Comment by albilu** [comment]
> Hi @albilu , could you give the snapshot release a try?

@bodiam works as expected thanks! any date on when it will be release?

**Comment by bodiam** [comment]
> > Hi @albilu , could you give the snapshot release a try?
> 
> @bodiam works as expected thanks! any date on when it will be release?

It's released, you can use the 2.0.2 now.
